src/main/java/org/rsa/net/apis/APIFactory.java

```diff
@@ -0,0 +1,52 @@
+package org.rsa.net.apis;
+
+import com.google.common.util.concurrent.RateLimiter;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import okhttp3.OkHttpClient;
+import org.rsa.net.HttpClient;
+import org.rsa.net.apis.discourse.DiscourseAPI;
+import org.rsa.net.apis.discourse.transformers.CategoryTransformer;
+import org.rsa.net.apis.discourse.transformers.TopicTransformer;
+import org.rsa.net.apis.roblox.RobloxAPI;
+import org.rsa.net.apis.wandbox.WandboxAPI;
+
+public class APIFactory {
```

Comment:
Can we create a method for serving `OkHttpClient`, `Gson` and `HttpClient`? I don't think we should have the logic duplicated across lines if not needed.